ai_flow/task_executor/kubernetes/pod_template_file_sample/pod_template.yaml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: base
+spec:
+  containers:
+    - args: []
+      command: []
+      env: []
+      envFrom: []
+      imagePullPolicy: IfNotPresent
+      name: worker
+      ports: []
+      volumeMounts:
+        - mountPath: "/opt/aiflow/logs"
```

Comment:
How to get the log.

Reply:
Pod is not deleted after finished, so we can get logs by accessing pods.